framework-cloud/framework-openshift/src/main/java/org/kie/cloud/openshift/deployment/KieServerDeploymentImpl.java

```diff
@@ -90,4 +99,44 @@ public String getServiceName() {
             RouterUtil.waitForRouter(getUrl());
         }
     }
+
+    @Override
+    public void waitForContainerRespin() {
+        // Expecting the config map name to be same as service name. Needs to be adjusted in case this changes!!!!
+        ConfigMap kieServerDeployments = getOpenShift().getConfigMap(getServiceName());
```

Comment:
Isn't there a label service name in the config map? We could find the config map by this label then.